### ISSUE TITLE
Skipping lib2bit.a and using only lib2bit.so? [Documentation issue mostly I think]

### ISSUE THREAD

**Issue by rubyFeedback** [dpryan79/lib2bit#7]
Hey there,

Right now when doing "make" and "make install" we instlal both
lib2bit.a and lib2bit.so. I am not sure I will need the lib2bit.a
though. Would there be a possibility to add a flag to the 
Makefile, e. g. to skip building the .a file or remove it?

It's not a big issue - I can manually remove the .a file or 
do so via scripts (e. g. ruby compiling lib2bit for me,
and then at post-installation remove the lib2bit.a file).

Rationale: Personally I try to use .so files when possible,
unless it is a statically compiled program. I have had 
some issues with .a files before, in particular "relocation
errors" due to -fPIC, such as in bzip2.a or similar files. But
as said, it's no big deal, I can remove it just fine after 
make install.

**Comment by dpryan79** [comment]
Yeah, we can probably make a `make dynamic` or something like that target.
